tests/integration/api/v1/curated_recommendations/test_curated_recommendations.py

```diff
@@ -374,39 +373,171 @@ async def test_curated_recommendations_topics(self, topics):
+                Topic.GAMING,
+                Topic.SPORTS,
+                Topic.SCIENCE,
+                Topic.SELF_IMPROVEMENT,
+            ],
+            [
+                Topic.EDUCATION,
+                Topic.PERSONAL_FINANCE,
+                Topic.BUSINESS,
+                Topic.TECHNOLOGY,
+                Topic.TRAVEL,
+                Topic.FOOD,
+                Topic.ARTS,
+                Topic.POLITICS,
+                Topic.GAMING,
+                Topic.SPORTS,
+                Topic.SCIENCE,
+                Topic.SELF_IMPROVEMENT,
+                Topic.PARENTING,
+            ],
+            [
+                Topic.EDUCATION,
+                Topic.PERSONAL_FINANCE,
+                Topic.BUSINESS,
+                Topic.TECHNOLOGY,
+                Topic.TRAVEL,
+                Topic.FOOD,
+                Topic.ARTS,
+                Topic.POLITICS,
+                Topic.GAMING,
+                Topic.SPORTS,
+                Topic.SCIENCE,
+                Topic.SELF_IMPROVEMENT,
+                Topic.PARENTING,
+                Topic.CAREER,
+            ],
+            [
+                Topic.EDUCATION,
+                Topic.PERSONAL_FINANCE,
+                Topic.BUSINESS,
+                Topic.TECHNOLOGY,
+                Topic.TRAVEL,
+                Topic.FOOD,
+                Topic.ARTS,
+                Topic.POLITICS,
+                Topic.GAMING,
+                Topic.SPORTS,
+                Topic.SCIENCE,
+                Topic.SELF_IMPROVEMENT,
+                Topic.PARENTING,
+                Topic.CAREER,
+                Topic.HEALTH_FITNESS,
+            ],
+        ],
+    )
+    async def test_curated_recommendations_preferred_topic(self, preferred_topics):
+        """Test the curated recommendations endpoint accepts 1-15 preferred topics &
+        top N recommendations contain the preferred topics.
+        """
         async with AsyncClient(app=app, base_url="http://test") as ac:
             response = await ac.post(
-                "/api/v1/curated-recommendations", json={"locale": "en-US", "topics": ["health"]}
+                "/api/v1/curated-recommendations",
+                json={"locale": "en-US", "topics": preferred_topics},
             )
             data = response.json()
             corpus_items = data["data"]
 
             assert response.status_code == 200
             # assert total of 80 items returned
             assert len(corpus_items) == 80
-            # assert boost_preferred_topic was called
-            boost_preferred_topic_spy.assert_called_once()
 
-            # extract IDs from both lists
-            fixture_ids = [
-                item["id"] for item in fixture_response_data["data"]["scheduledSurface"]["items"]
-            ]
-            corpus_ids = [item["scheduledCorpusItemId"] for item in corpus_items]
-            # assert that recs were reordered
-            assert fixture_ids != corpus_ids
+            # get the top N recs
```

Comment:
The next line doesn't get recs. Would the following be more accurate?
```suggestion
            # determine the number of recommendations that are expected to be preferred
```